### DISCUSSION TITLE
Add numeric outline to headers (if wanted)

### DISCUSSION THREAD

**Discussion by ls-c0de** [l1xnan/obsidian-better-export-pdf#345] [Ideas]
What about the option that each Headline gets assigned a numeric outline (If you want it), so it would look like this:

.md File
---------------------
```
## Intro
## Main Part
### Breaking free of the Matrix
#### Free your Mind
#### There is no Spoon
### Destroying the Matrix
#### Destroying Smith
#### Unleashing Total Power
## The End?
```
---------------------

So, in the End on the pdf it would look like this:

---------------------
1. Intro
2. Main Part
2.1 Free your Mind
2.2 There is no spoon
3. Destroying the Matrix
3.1 Destroying Smith
3.2 Unleashing Total Power
4. The End?
---------------------

The advantage is, that you can easily add new sub points without changing anything before or after. You only need to pay attention to the structure itself, and can easily decide whether you want a outline or not.

**Comment by l1xnan** [comment]
[Number Headings](https://github.com/onlyafly/number-headings-obsidian) plugin or [Header counter](https://forum.obsidian.md/t/header-counter/3525/57)?

**Reply by ls-c0de** [reply]
Thank you very much, didn't thought about just looking it up haha
